client/components/site-selector/style.scss

```diff
@@ -87,6 +88,7 @@
 .site-selector__sites {
 	max-height: calc( 100% - 89px );
 	overflow-y: auto;
+	border-top: 1px solid lighten( $gray, 20% );
```

Comment:
This would break when search is not present. The border should be applied to the search element.